packages/ag-charts-community/src/chart/series/polar/polarSeries.ts

```diff
@@ -1,11 +1,16 @@
 import type { ModuleContext } from '../../../module/moduleContext';
+import { StateMachine } from '../../../motion/states';
 import type { BBox } from '../../../scene/bbox';
 import type { PointLabelDatum } from '../../../util/labelPlacement';
 import { ChartAxisDirection } from '../../chartAxisDirection';
 import type { DataModel, ProcessedData } from '../../data/dataModel';
 import type { SeriesNodeDataContext, SeriesNodeDatum } from '../series';
 import { Series, SeriesNodePickMode } from '../series';
 
+type PolarAnimationState = 'empty' | 'ready' | 'waiting' | 'clearing';
+type PolarAnimationEvent = 'update' | 'updateData' | 'clear';
+type PolarAnimationData = { duration: number };
```

Comment:
Should be optional `duration?: number`, it's only used when `animateClearingUpdateEmpty()` calls `this.animationState.transition('update', { duration: updateDuration });`

Reply:
Fixed